tests/phpunit/constantsTest.php

```diff
@@ -42,6 +42,24 @@ public function testConstantsDefined() {
     $this->assertEquals($text, $expanded->parsed_text());
   }
   
+  public function testImplicitConstants() {
+    // Consonants
+    $this->assertEquals('X', title_capitalization('x', TRUE));
+    $this->assertEquals('Xz', title_capitalization('xz', TRUE));
+    $this->assertEquals('XZZ', title_capitalization('xzz', TRUE));
```

Comment:
Combining texts also allows us to check that sequences work correctly
```suggestion
    $this->assertEquals('XZZ BBBB/EEE', title_capitalization('xzz bbbb/eee', TRUE));
```